src/data/polynomial/monomial.lean

```diff
@@ -37,6 +38,14 @@ lemma C_mul : C (a * b) = C a * C b := C.map_mul a b
 
 lemma C_add : C (a + b) = C a + C b := C.map_add a b
 
+lemma C_sum {α : Type u} (s : finset α) (f : α → R) :
+C (∑ i in s, f i) = ∑ i in s, C (f i) :=
+(finset.sum_hom s (λ x, C x)).symm
```

Comment:
I think this exists already as `C.map_sum`?

Reply:
Yup, hopefully you can just use `C.map_sum` at the place where you need it.